src/frontend/mod.rs

```diff
@@ -718,10 +728,36 @@ impl<'a> Frontend<'a> {
         &mut self, WithRange(alignment, range): WithRange<Vec<Alignment>>, cskvp: Option<Cskvp<'a>>,
     ) {
         let mut cskvp = cskvp.unwrap_or_default();
+
+        let mut column_lines = vec![Vec::new(); alignment.len()];
+        loop {
+            match &self.parser.peek().unwrap().0 {
+                &CmarkEvent::Start(CmarkTag::TableHead) | &CmarkEvent::Start(CmarkTag::TableRow) => {
+                    let mut i = 0;
+                    loop {
+                        let cell_text = match &self.parser.peek().unwrap().0 {
+                            &CmarkEvent::Start(CmarkTag::TableCell) => self.concat_until_end_inclusive(false),
+                            &CmarkEvent::End(CmarkTag::TableHead) | &CmarkEvent::End(CmarkTag::TableRow) => break,
+                            _ => unreachable!(),
```

Comment:
Can you add a comments to these unreachables / some message?
Not necessarily for the current program, I think they are correct, but just in case common mark or our parsing style changes and these turn into bugs.

Reply:
Done. Good idea.